exchange-contract/README.md

```diff
@@ -0,0 +1,92 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+**The protocols and software are for reference purposes only and not intended for production usage.**
+
+# Exchange Contract Family
+
+The Exchange contract family is a suite of contracts that demonstrate many of the capabilities of the private data objects technologies.
+
+Three basic contracts define the elements of the Exchange contract family: the asset type contract, the vetting organization contract, and the issuer contract. Three additional contracts extend the Exchange contract family for trading potentially confidential assets using non-fungible tokens.
+
+## Contracts for Traditional Assets
+
+An *asset type* contract object defines a unique, shared identifier for a type of digital asset and a schema for the representation for assets of that type. For example, we might define an asset type contract object for blue marbles. Since the identifier for the contract object is unique, it provides a shared, unique identifier that can be used to refer to assets that are blue marbles.
+
+A *vetting organization* contract object manages a list of contract objects authorized to issue assets of a particular type. While actual vetting of an issuer occurs outside the contract object, the object provides a means of recording the decision to authorize an issuer. In this way, the vetting organization contract object provides a root of trust for issuers of a particular asset type. Continuing the blue marble example, the Blue Marble Players Association might create a vetting organization object to record the identities of local chapters that may issue blue marble holdings to their members.
+
+The *issuer* contract object maintains a balance sheet that captures ownership of assets of a particular type. The issuer contract allows the creator of the contract object to issue assets (that is, assign ownership of assets to a particular individual). However, once the initial issuance occurs, confidentiality of transactions and balances is maintained; in this case, even the creator of the issuer contract object is not granted the right to examine the quantity of assets owned by an individual after the initial issuance. Local chapters of the Blue Marble Players Association each
```

Comment:
imho, instead of saying that the "issuer issues assets", it would be more appropriate to say issuer issues "tokens to assets". Assets could be real assets and have a meaning/utility beyond what can be captured by a pdo. 

Further, we should also clarify that the binding the asset and the token is out of band, and is beyond the scope of the issuer contract.

Reply:
actually... issuers issue assets. token issuers issue tokens.